lib/Simplifier/AIGSimplifyPropositionalCore.cpp

```diff
@@ -210,11 +213,11 @@ ASTNode AIGSimplifyPropositionalCore::topLevel(const ASTNode& top)
     assert((it->second).size() == 1); // should be a propositional variable.
     const int index =
         (it->second)[0].symbol_index; // This is the index of the pi.
-    Aig_Obj_t* pi = Aig_ManPi(mgr.aigMgr, index);
+    Aig_Obj_t* pi = Aig_ManLi(mgr.aigMgr, index);
     ptrToOrig.insert(make_pair(pi, result));
   }
 
-  Aig_Obj_t* pObj = (Aig_Obj_t*)Vec_PtrEntry(mgr.aigMgr->vPos, 0);
+  Aig_Obj_t* pObj = (Aig_Obj_t*)Vec_PtrEntry(mgr.aigMgr->vCos, 0);
```

Comment:
The static analyzer complained about this, but it isn't a regression so I don't think you need to care.